tasks.py

```diff
@@ -32,9 +32,9 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_ansible": {
-            "nautobot_ver": "2.0.0",
+            "nautobot_ver": "2.3.5",
```

Comment:
Please revert the changes to the versions in `tasks.py`. If you want to override your local environment, copy the `invoke.yml.example` at the root of the project to `invoke.yml` and uncomment and update the values you want there instead.